src/karma/KarmaParser.ts

```diff
@@ -66,6 +74,10 @@ export class KarmaParser {
             requestedChange *= -1;
         }
 
+        if (firstCharacter === USER_DESIGNATOR) {
```

Comment:
IMO we should in-line `firstCharacter`, it's only used here and it's declared pretty far from its only use. This is also a good place for a short comment mentioning that this is the formatting needed by Slack. (It's clear what the code is doing, but not why)